config/environments/development.rb

```diff
@@ -41,7 +41,7 @@
 
   config.after_initialize do
     Bullet.enable = true
-    Bullet.alert = true
+    Bullet.alert = false
```

Comment:
I find the alerts sorta annoying and reading the logs or clicking in the bottom left to work just fine. 🤷🏼‍♂️

Reply:
Fine with me